database/factories/Order.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Faker\Generator as Faker;
+
+use Ramsey\Uuid\Uuid;
+
+$factory->define(App\Order::class, function (Faker $faker) {
+    return [
+        'uuid' => Uuid::uuid4()->toString(),
```

Comment:
Will we still be using this ? Or is this simply out of the scope of this particular PR ?

Reply:
The latter; I'll swap that for a `user_id`, as discussed in a side-channel, in a subsequent PR.